### PR TITLE
Modified DataServlet to create a history of comments

### PR DESCRIPTION
**Pull request by thuavu**
The DataServlet is modified so that the comments are stored in a Servlet, thus instead of storing hard code comment, it takes comments from the form add it to the history. The history of comments is then displayed at the bottom of the form page.

### CODE REVIEW

**Comment by thuavu** [comment]
image

**Comment by joanapascoal** [comment]
Can you please add a description about what is being changed here?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -20,23 +20,43 @@
...
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+
+    // Add text from the form to the comments ArrayList
+    comments.add(text);
+
+    // Print in console to test
+    System.out.println(text);
```
Comment (joanapascoal) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Avoid keep System.out.println on the submitted code. They are a great help during development, but in the final code that avoid the code cleanness and readability.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -20,23 +20,43 @@
...
     String json = gson.toJson(comments);
     response.getWriter().println(json);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+
+    // Add text from the form to the comments ArrayList
```
Comment (joanapascoal) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: dot is missing.

**Review by joanapascoal — APPROVED** [review]
LGTM.

Just a few minor comments and please be add the the PR description.

**Comment by thuavu** [comment]
> Can you please add a description about what is being changed here?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (thuavu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Will keep that in mind

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (thuavu) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: @joanapascoal Can you clarify what dot was missing exactly?

**Pull request merged**